templates/jlink/jlink-builder/jlink-builder-template.yaml

```diff
@@ -29,7 +29,15 @@ objects:
       dockerfile: |
         FROM -
         USER 0
-        RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-${JDK_VERSION}-openjdk-jmods
```

Comment:
This line shouldn't be changed: we need the jmods RPM to be installed in this container in the proper location.